src/auth/connection.ts

```diff
@@ -28,6 +28,7 @@ export const isIamConnection = (conn?: Connection): conn is IamConnection => con
 export const isSsoConnection = (conn?: Connection): conn is SsoConnection => conn?.type === 'sso'
 export const isBuilderIdConnection = (conn?: Connection): conn is SsoConnection =>
     isSsoConnection(conn) && conn.startUrl === builderIdStartUrl
+export const isEnterpriseSsoConnection = (conn?: Connection) => isSsoConnection(conn) && !isBuilderIdConnection(conn)
```

Comment:
If we just did the inverse of `isBuilderIdConnection` it could imply enterprise sso or IAM credentials.
This is why we need the granularity of this function for non-builder id sso connections.

Reply:
I saw "enterprise" being used in the CodeWhisperer code, so I assumed that this is the new standard.

Instead of "enterprise" I could also see using "identityCenter".
So we would have:
-  "sso" that could be Builder ID or Identity Center
- "builderId" which is only builder id sso's
- "identityCenter" which is only identityCenter sso's

Reply:
`identityCenter` or `Idc` is better. "enterprise" is specific to CW's particular currently supported concepts. Their functions should also be renamed, but that should be a separate PR.